app/components/Footer/default.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { PropTypes } from 'react';
+
+
+const Footer = props =>
+  <footer className="main">
+    <div className="credits" title="Well, no there is nothing special here">
```

Comment:
I think a punctuation is missing in the title: `Well, no: there is nothing special here!`